Server/service/networkService.js

```diff
@@ -144,42 +151,78 @@ class NetworkService {
       const response = await axios.get(job.data.url);
       return response;
     };
-
+  
     let isAlive;
-
-    // attempt connection
+  
     try {
-      const { responseTime, response } =
-        await this.measureResponseTime(operation);
-      // check if response is in the 200 range, if so, service is up
+      // Measure response time and get the response
+      const { responseTime, response } = await this.measureResponseTime(operation);
       isAlive = response.status >= 200 && response.status < 300;
-
-      //Create a check with relevant data
+  
+      // Fetch monitor TTL
+      const monitor = await this.db.getMonitorById(job.data._id);
```

Comment:
We don't need to do this query, `job.data` is this `Monitor`.  You can revert this code and just add the TTL parts

Reply:
Alrighty sounds good. Will do! Thanks!!